ledger/ledger-api-test-tool/BUILD.bazel

```diff
@@ -148,8 +148,10 @@ EOF
                 "//ledger/test-common:semantic-tests%s.scala" % lf_version_suffix,
                 "//libs-scala/grpc-utils",
                 "//libs-scala/timer-utils",
-                "@maven//:ai_x_diff_2_12",
                 "@maven//:com_chuusai_shapeless_2_12",
+                "@maven//:com_propensive_magnolia_2_12",
+                "@maven//:com_propensive_mercator_2_12",
```

Comment:
While these libraries are not directly used in our code, they are transitive dependencies of the `diffx`. Without them being explicitly listed Bazel will not put them on the classpath and automatic type class derivation will not work.

Reply:
Perhaps add a comment to the Bazel file as such?

Reply:
I think at this point everybody working with Bazel is already familiar with the fact that transitive dependencies must be listed explicitly.

Reply:
the errors here look pretty different from the ones you get usually so I think it’s worth adding a comment even for people familiar with Bazel.

Reply:
That, and so I know exactly what's depending on them. Something like `// transitive dependency of diffx` would help me a lot.

Reply:
I've added comments for both `magnolia` and `mercator`. As @cocreature mentioned errors in this case were rather misleading, especially for the `mercator`.